File: src/Suframa.Sciex.UserInterface/www/app/view-model/FiltroCadastroPessoaJuridicaVM.ts
import { naturezaQualificacaoVM } from './NaturezaQualificacaoVM';

export class FiltroCadastroPessoaJuridicaVM {
	constructor() {
		this.naturezaQualificacao = new Array<naturezaQualificacaoVM>();
	}

	cnpj: string;
	idNaturezaGrupo?: number;
	idNaturezaJuridica?: number;
	idPessoaJuridica?: number;
	idRequerimento?: number;
	isQuadroSocietario: boolean;
	naturezaQualificacao: naturezaQualificacaoVM[];
	tipoEntidadeRegistro: number;
	tipoEstabelecimento: number;
}

File: src/Suframa.Sciex.CrossCutting.DataTransferObject/ViewModel/FiltroCadastroPessoaJuridicaVM.cs
using Suframa.Sciex.CrossCutting.DataTransferObject.Enum;
using System.Collections.Generic;

namespace Suframa.Sciex.CrossCutting.DataTransferObject.ViewModel
{
	public class FiltroCadastroPessoaJuridicaVM
	{
		public string Cnpj { get; set; }
		public int? IdNaturezaGrupo { get; set; }
		public int? IdNaturezaJuridica { get; set; }
		public int? IdPessoaJuridica { get; set; }
		public int? IdRequerimento { get; set; }
		public bool IsQuadroSocietario { get; set; }
		public IEnumerable<NaturezaQualificacaoVM> NaturezaQualificacao { get; set; }
		public EnumTipoEntidadeRegistro TipoEntidadeRegistro { get; set; }
		public EnumTipoEstabelecimento TipoEstabelecimento { get; set; }
	}
}